svelte/src/app.d.ts

```diff
@@ -0,0 +1,13 @@
+// See https://kit.svelte.dev/docs/types#app
```

Comment:
I followed the link, but still don't understand where it's being used in the application. Is it required for the library or for the sample application?  In the first case we should add it into the exported public interface in the later move it into the example itself.

Reply:
This file would be for the sample application example. The library itself does not use it. Vite could use the file when bundling into a package